schema.sql

```diff
@@ -4,5 +4,6 @@ CREATE TABLE animals(
   date_of_birth DATE NOT NULL,
   espcape_attempts INT NOT NULL,
   neutered BOOLEAN NOT NULL,
-  weight_kg DECIMAL (5, 2) NOT NULL
+  weight_kg DECIMAL (5, 2) NOT NULL,
+  species VARCHAR (255) NOT NULL
```

Comment:
- [ ] I will strongly recommend you dont modify the table directly, but rather alter the table by adding the species column by running the command:

`ALTER TABLE animals ADD species varchar(255);`